### PR TITLE
Add Dockerfile to build tidb-server

### PR DESCRIPTION
**Pull request by netroby**
You must install docker first, then

docker build --rm -t tidb-server .

after build success, you may see images via

docker images

docker run -d --name tidb-server tidb-server

Signed-off-by: ZhiFeng Hu <hufeng1987@gmail.com>

### CODE REVIEW

**Comment by netroby** [comment]
#522

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM jeanblanchard/alpine-glibc
```
Comment (siddontang) [inline Dockerfile]: why not use golang 1.5 docker image directly?
See https://github.com/docker-library/docs/tree/master/golang

Comment (netroby) [inline Dockerfile]: image size should not too big.  using alpine glibc ,the image size just 53MB


通过 Android 版 Outlook 发送





On Thu, Nov 5, 2015 at 5:39 AM -0800, "siddontang" <notifications@github.com> wrote:












In Dockerfile:
> @@ -0,0 +1,22 @@
> +FROM jeanblanchard/alpine-glibc



why not use golang 1.5 docker image directly?

See https://github.com/docker-library/docs/tree/master/golang

**Comment by siddontang** [comment]
Hi @netroby , the docker environment should be the same as production environment, so we should use official golang image or popular ubuntu/debian/fedora, etc.
We don't care image size too much.

**Comment by netroby** [comment]
The container does not included Golang environment, it just build then generate binary executable tidb-server. then we can running the server inside container.

It's no need to include Golang environment at all, because the server ready for serve, no need recompile or include source code. 
If we need recompile. The Dockfile can be used to compile it . generate new container.

**Comment by ngaut** [comment]
Hi, @netroby :
 Could you use golang as default image ? 
It would be more convenient to run it, test it or even try to patch it. Thanks

**Comment by netroby** [comment]
Ok

**Pull request closed without merging**